### PR TITLE
740 qplot warning failure@main

### LINKED ISSUE #740: Integration test failure
```
  ✖ | 1       2 | g_km [1.3s]                                                     
  ────────────────────────────────────────────────────────────────────────────────
  Failure (test-g_km.R:8:3): g_km default plot works
  `g_km(df = df, variables = variables, ci_ribbon = FALSE, draw = FALSE)` produced warnings.
  ────────────────────────────────────────────────────────────────────────────────
```

Also:
```
✔ |   1     6 | decorate_grob [1.1s]                                            
  ────────────────────────────────────────────────────────────────────────────────
  Warning (test-decorate_grob.R:63:3): decorate_grob_set returns no warnings when creating a non-empty plot
  `qplot()` was deprecated in ggplot2 3.4.0.
  Backtrace:
    1. base::with(...)
         at test-decorate_grob.R:63:2
    2. base::with.default(...)
    3. base::eval(substitute(expr), data, enclos = parent.frame())
    4. base::eval(substitute(expr), data, enclos = parent.frame())
   12. ggplot2::qplot(Sepal.Length, Sepal.Width, col = Species)
   13. ggplot2:::deprecate_soft0("3.4.0", "qplot()")
  ────────────────────────────────────────────────────────────────────────────────
```
and
```
 ✔ |   1     1 | wrap_text                                                       
  ────────────────────────────────────────────────────────────────────────────────
  Warning (test-wrap_text.R:3:3): wrap_text works with default settings
  This is a deprecated function as there is a current implementation in rtables, not based on grid::
  Backtrace:
   1. tern:::wrap_text(txt = text, width = grid::unit(4, "cm"), collapse = "\n")
        at test-wrap_text.R:3:2
  ────────────────────────────────────────────────────────────────────────────────
```

### PR DESCRIPTION
**Pull request by Melkiades**
Fixes #740

The error was actually originating from:
```
── Warning (test-g_km.R:8): g_km default plot works ────────────────────────────
The `size` argument of `element_line()` is deprecated as of ggplot2 3.4.0.
ℹ Please use the `linewidth` argument instead.
```

Interestingly, I also found the following which will be a dedicated issue:
```
── Warning (test-g_km.R:8): g_km default plot works ────────────────────────────
`aes_string()` was deprecated in ggplot2 3.0.0.
ℹ Please use tidy evaluation ideoms with `aes()`
```

### CODE REVIEW

**Comment by Melkiades** [comment]
There might be a problem with the first warning (i.e. it will be present if we have version 3.4.0 of ggplot2) as here it fails when it is given version 3.6

File: tests/testthat/test-wrap_text.R [hunk 1]
```diff
@@ -1,6 +1,8 @@
 testthat::test_that("wrap_text works with default settings", {
   text <- "This is a test with many words and more"
-  result <- wrap_text(txt = text, width = grid::unit(4, "cm"), collapse = "\n")
+  result <- testthat::expect_warning(
```
Comment (nikolas-burkoff) [inline tests/testthat/test-wrap_text.R:3]: so you can do this https://lifecycle.r-lib.org/reference/verbosity.html#ref-examples

Comment (Melkiades) [reply tests/testthat/test-wrap_text.R]: thanks! Modified accordingly

**Review by edelarua — APPROVED** [review]
Lgtm @Melkiades!

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks @Melkiades

**Pull request merged**